plugins/collapseGroups.js

```diff
@@ -68,11 +71,17 @@ export const fn = () => {
           node.children.length === 1
         ) {
           const firstChild = node.children[0];
+          let nodeHasFilter = !!node.attributes.filter;
+          if (!nodeHasFilter && node.attributes.style) {
+            if (computeStyle(stylesheet, node)['filter']) {
+              nodeHasFilter = true;
+            }
+          }
```

Comment:
```suggestion
          const nodeHasFilter = !!(
            node.attributes.filter || computeStyle(stylesheet, node).filter
          );
```

Now that you're using `computeStyle`, and considering that it can have a `filter` applied without being in the `style` attribute, you can drop that check for if the `style` attribute is defined.

I think it's simplest to stick a `const` and do both checks in the same line. It should mean the exact same thing as what you have now.